src/main/java/uk/gov/hmcts/reform/userprofileapi/domain/feign/IdamFeignClient.java

```diff
@@ -53,4 +59,35 @@ public interface IdamFeignClient {
     @Headers({"Authorization: {authorization}", "Content-Type: application/json"})
     public Response deleteUser(@PathVariable("userId") String userId);
 
+    @GetMapping(value = "/api/v1/users", consumes = {"application/x-www-form-urlencoded"})
+    @Headers("authorization: {authorization}")
+    public Response getUserFeed(@RequestParam Map<String, String> params);
+
+    @Data
+    class User {

```

Comment:
is there any reason to create User in feignclient? if not its better to move it to new class